File: 1313_decompress_run.js
var decompressRLElist = function(nums) {
    let i= 0;
    let ans= [];
    while(i < nums.length){
        let firstEle = nums[i];
        let secondEle = nums[i+1]
        for(let j = 0; j < firstEle; j++){
            ans.push(secondEle);
        }
        i += 2;
    }
    return ans;
};

File: 1313_decompress_run.rb
def decompress_rl_elist(nums)
    i= 0;
    ans= [];
    while i < nums.length do
        firstEle = nums[i];
        secondEle = nums[i+1]
        (0...firstEle).each do |j|
            ans.push(secondEle);   
        end
        i += 2;
    end
    return ans;
end